src/controllers/appointments.js

```diff
@@ -0,0 +1,11 @@
+const { Appointment } = require('../models/appointments');
+
+const create = async (req, res) => {
+	const appointment = await Appointment.create(req.body);

```

Comment:
It's business-logic, use service for it.
I pretty much recommend you read this [topic](https://www.codementor.io/@evanbechtol/node-service-oriented-architecture-12vjt9zs9i).

Reply:
Ok, I will read it and will separate this logic to service